docs/advanced/document-store/benchmark.md

```diff
@@ -169,7 +336,7 @@ and report quality measurement (`Recall@K`).
 * **Not just speed, but also quality**: Since some backends offer better performance and some offer better quality, we make sure to report the quality measurement for the approximate nearest neighbor search. This will allow users to 
```

Comment:
I think we should say here that we show the trade-off between quality and speed as you tune your parameters in each document store